cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -79,23 +87,24 @@ GROUP BY pws_id, geom;
 -- Parcel-level data
 
 CREATE TABLE IF NOT EXISTS parcels (
-    id serial PRIMARY KEY,
     -- TODO: consider standardizing addresses into a PostGIS type once we have data.
     -- https://postgis.net/docs/manual-2.5/Address_Standardizer.html
     address text,
-    sl_path geometry(Geometry, 4326),
-    lead_prediction float
-);
-ALTER TABLE parcels
```

Comment:
Put this all into CREATE TABLE statement since this table needs to be dropped / renamed anyway